README.md

```diff
@@ -45,6 +45,12 @@ Clone or fork the Heptio Ark repo:
 git clone git@github.com:heptio/ark.git
 ```
 
+Checkout the appropriate tag for the Ark version you wish to use (e.g. replacing `<VERSION TAG>` with "v0.3.3"):
+```
+git checkout <VERSION TAG>
+```
+You should `git checkout master` only if you're planning on [building the Ark image from scratch][7].
```

Comment:
I don't want to discourage people from building. I'm just highlighting an issue that we have if you are trying to build and test the image locally.

For end-users, they should ideally deploy a specific `@sha256` image id or a specific versioned tag (`v0.3.3`). But...we have the deployment yaml using `latest` so we don't have to remember to bump the version in there each time we do a new release, but that makes local testing a bit harder. For us developers, we need to be able to easily build and deploy an image for testing, ideally without having to remember to change the `imagePullPolicy`.

I'm not sure yet the best way to fix this for both users and devs. @skriss any thoughts?